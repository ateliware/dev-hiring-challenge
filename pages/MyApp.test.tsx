import { render } from "@testing-library/react";

import MyApp from "./index.page";

describe("MyApp", () => {
  it("renderiza o app", async () => {
    const wrapper = render(<MyApp />);
    expect(wrapper.container).toMatchSnapshot();
  });
});
