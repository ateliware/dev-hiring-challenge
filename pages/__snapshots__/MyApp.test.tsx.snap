// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyApp renderiza o app 1`] = `
<div>
  <div>
    <header
      class="header"
    >
      <h2
        class="title"
        role="heading"
      >
        Repositórios
      </h2>
      <p
        class="searchingTerm"
        data-testid="searchTerm"
      >
        pesquisando 
        <strong />
         em nome e descrição
      </p>
      <form
        class="formContainer"
      >
        <input
          name="pesquisa"
          placeholder="pesquisa"
          value=""
        />
        <button
          type="submit"
        >
          pesquisar
        </button>
      </form>
    </header>
    <main
      class="repositioriesListsContainer"
    >
      <div
        class="repositoryCategoryContainer"
      >
        <article
          class="main"
        >
          <header
            class="header"
          >
            <h3
              class="title"
            >
              PHP
            </h3>
            <p
              class="lead"
            >
              Mostrando 0 repósitorios, ordenados por quantidade de estrelas do servidor MySQL local atualizado em 
            </p>
            <button
              class="refreshDataButton"
              disabled=""
            >
              Atualizar banco de dados desde a API de Github
            </button>
          </header>
          <div
            class="refreshingIndicator"
          >
            carregando...
          </div>
        </article>
      </div>
      <div
        class="repositoryCategoryContainer"
      >
        <article
          class="main"
        >
          <header
            class="header"
          >
            <h3
              class="title"
            >
              Typescript
            </h3>
            <p
              class="lead"
            >
              Mostrando 0 repósitorios, ordenados por quantidade de estrelas do servidor MySQL local atualizado em 
            </p>
            <button
              class="refreshDataButton"
              disabled=""
            >
              Atualizar banco de dados desde a API de Github
            </button>
          </header>
          <div
            class="refreshingIndicator"
          >
            carregando...
          </div>
        </article>
      </div>
      <div
        class="repositoryCategoryContainer"
      >
        <article
          class="main"
        >
          <header
            class="header"
          >
            <h3
              class="title"
            >
              Javascript
            </h3>
            <p
              class="lead"
            >
              Mostrando 0 repósitorios, ordenados por quantidade de estrelas do servidor MySQL local atualizado em 
            </p>
            <button
              class="refreshDataButton"
              disabled=""
            >
              Atualizar banco de dados desde a API de Github
            </button>
          </header>
          <div
            class="refreshingIndicator"
          >
            carregando...
          </div>
        </article>
      </div>
      <div
        class="repositoryCategoryContainer"
      >
        <article
          class="main"
        >
          <header
            class="header"
          >
            <h3
              class="title"
            >
              Java
            </h3>
            <p
              class="lead"
            >
              Mostrando 0 repósitorios, ordenados por quantidade de estrelas do servidor MySQL local atualizado em 
            </p>
            <button
              class="refreshDataButton"
              disabled=""
            >
              Atualizar banco de dados desde a API de Github
            </button>
          </header>
          <div
            class="refreshingIndicator"
          >
            carregando...
          </div>
        </article>
      </div>
      <div
        class="repositoryCategoryContainer"
      >
        <article
          class="main"
        >
          <header
            class="header"
          >
            <h3
              class="title"
            >
              Dart
            </h3>
            <p
              class="lead"
            >
              Mostrando 0 repósitorios, ordenados por quantidade de estrelas do servidor MySQL local atualizado em 
            </p>
            <button
              class="refreshDataButton"
              disabled=""
            >
              Atualizar banco de dados desde a API de Github
            </button>
          </header>
          <div
            class="refreshingIndicator"
          >
            carregando...
          </div>
        </article>
      </div>
    </main>
  </div>
</div>
`;
